build_tools/ci/cpu_comparison/run_test.sh

```diff
@@ -184,52 +222,68 @@ function run_test() {
 
   set -x
 
-  # Assert that file_name is set:
+  if [ -n "${test_file}" ]; then
+    # Update test_file to be a complete path to the file
+    test_file=$(realpath "${test_file}")
+
+    # Now test_file is a string of the /path/to/name.mlir.
+    # Confirm this with clear error messages, and extract 'name' as a variable
+    name=$(basename "${test_file}" .mlir)
+
+    # Running 'python3 ${INPUT_GENERATOR} ${test_file} ${OUTPUT_DIR}' does 2 things.
+    # 1) it creates binary files with the data that will be consumed as inputs
+    #    by iree-run-module
+    # 2) it prints a line with the names of the binary files, which
+    #    iree-run-module will have appended (input and output flags).
+    #
+    python3 ${INPUT_GENERATOR} ${test_file} ${OUTPUT_DIR}
+  else
+    # Generate a name for the test based on name_prefix and the matmul dimensions
+    # and data types, assuming m, n, k are just single integers:
+    name="${name_prefix}_${m}x${n}x${k}_${lhs_rhs_type}_${acc_type}"
+
+    # Running 'python3 ${INPUT_GENERATOR} ${test_file} ${OUTPUT_DIR}' does 3 things.
+    # 1) it creates a mlir file using the test template;
+    # 2) it creates binary files with the data that will be consumed as inputs
+    #    by iree-run-module;
+    # 3) it prints a line with the names of the binary files, which
+    #    iree-run-module will have appended (input and output flags).
+    #
+    python3 ${INPUT_GENERATOR} ${OUTPUT_DIR} ${name_prefix} ${m} ${n} ${k} ${lhs_rhs_type} ${acc_type}
+
+    # Path of the test mlir and vmfb files
+    test_file="${OUTPUT_DIR}/${name}.mlir"
+  fi
+
+  # Assert that the test file is exist
   if [ -z "${test_file}" ]; then
     echo "The test file must be provided."
     exit 1
   fi
+  echo "**** Running test for ${test_file} ****"
 
-  # Update test_file to be a complete path to the file
-  test_file=$(realpath "${test_file}")
-
-  # Now test_file is a string of the /path/to/name.mlir.
-  # Confirm this with clear error messages, and extract 'name' as a variable
-  name=$(basename "${test_file}" .mlir)
-  echo "**** Running test for ${name} ****"
-
+  # Disable exit on failure:
+  set +e
```

Comment:
@yzhang93 the errors are not re-enabled again? This test is currently failing on main, if you remove this line for me with 

```
error: One or more operations with large vector sizes (8192 bytes) were found
```